interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncounterccdadispatchTable.php

```diff
@@ -728,7 +728,17 @@ public function getOfficeContact($pid, $encounter)
         if (empty($details)) {
             return '';
         } else {
-            $organization_uuid = UuidRegistry::uuidToString($details['facility_uuid']);
```

Comment:
Would it be easier to reduce the condition wrapping? This just keeps us from existing entirely in an if/then condition.

```php
$org_uuid = $details['facility_uuid'] ?? '';

if ($org_uuid == '') {
    // Process error
}
```